db/schema.rb

```diff
@@ -10,7 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_04_13_091115) do
+ActiveRecord::Schema.define(version: 2020_04_17_021537) do
```

Comment:
Was this file meant to be in this PR?

Reply:
hmm i'm not too sure because it's been updated every time i pull from master but i think in this specific PR we aren't changing anything in the schema so it shouldn't be updated?

Reply:
oh i think this may have taken some chances from your questionnaire refactoring PR. Let's remove these changes from this PR: https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request

Reply:
got it, just removed it from the pr - thanks for the catch :)